runtime/runtime-common/src/main/kotlin/pbandk/Message.kt

```diff
@@ -28,6 +28,13 @@ interface Message<T : Message<T>> {
         }
     }
 
+    abstract class OneOf<T> {
+        abstract val value: T
```

Comment:
🔧 Not blocking this PR, but you could store `value` in the abstract class itself rather than making the inheriting class store it:

```kt
abstract class OneOf<T>(val value: T) { ... }

sealed class Foo<V>(value: V) : OneOf<V>(value) {
    class Bar(bar: String = "") : Foo<String>(bar)
 }
```

Reply:
Done